README.md

```diff
@@ -67,3 +67,42 @@ This application is deployed at [pilot-2-submission-rhino](https://connect.appsi
 - No tests are added as of now.
 
 - No custom CSS/JS is added as of now.
+
+# Packing the code for FDA Submission
+
+Proprietary packages cannot be submitted as a compressed file through the eCTD gateway. However, proprietary R packages can be converted to a plain `.txt` file by `pkglite` then submitted through the eCTD gateway. FDA reviewers can use `pkglite` to convert the .txt file back to R package.
+We have to package our shiny app using the same package `pkglite` in order for the file to go through the eCTD gateway. Since the `pkglite` package was built to pack R packages to `.txt` file we have to create a simple DESCRIPTION file with just a package name to use `pkglite` to pack any R script into a `.txt` file.
```

Comment:
It improves readability for the edits to have phrases within the same paragraph to be in different lines.

I assume you wish to split the paragraph at `line 74`

```suggestion
Proprietary packages cannot be submitted as a compressed file through the eCTD gateway.
However, proprietary R packages can be converted to a plain `.txt` file by `pkglite` then submitted through the eCTD gateway.
FDA reviewers can use `pkglite` to convert the .txt file back to R package.

We have to package our shiny app using the same package `pkglite` in order for the file to go through the eCTD gateway.
Since the `pkglite` package was built to pack R packages to `.txt` file we have to create a simple DESCRIPTION file with just a package name to use `pkglite` to pack any R script into a `.txt` file.
```

Reply:
I would also rephrase the text a bit in line with BLUF approach.

1. First paragraph is affirmative (submission needs to be in plain text format, we recommend using pkglite)
2. Following paragraph(s) explains why we cannot use binary formats for compression and other necessary details
3. Last paragraph circles back to an overview of what needs to be done for a rhino application (connecting to next section)
    * As you already have 😁